roles/kube_prometheus_stack/meta/main.yml

```diff
@@ -29,3 +29,17 @@ dependencies:
     vars:
       upload_helm_chart_src: "{{ kube_prometheus_stack_helm_chart_path }}"
       upload_helm_chart_dest: "{{ kube_prometheus_stack_helm_chart_ref }}"
+  - role: keycloak_atmosphere_realm
```

Comment:
so we can technically remove this cause we can make `keycloak_client` depeend on `keycloak_realm` pointing to `keycloak_client_realm_id`